palisade-service/src/component-tests/java/uk/gov/gchq/palisade/component/palisade/CommonTestData.java

```diff
@@ -45,8 +45,8 @@
 
     public static final AuditErrorMessage AUDIT_ERROR_MESSAGE = AuditErrorMessage.Builder.create(PALISADE_REQUEST, new HashMap<>()).withError(ERROR);
 
-    public static final AuditablePalisadeRequest AUDITABLE_WITH_REQUEST = AuditablePalisadeRequest.Builder.create()
+    public static final AuditablePalisadeResponse AUDITABLE_WITH_REQUEST = AuditablePalisadeResponse.Builder.create()
             .withPalisadeRequest(PALISADE_REQUEST);
```

Comment:
```suggestion
            .withPalisadeResponse(PALISADE_REQUEST);
```
Might be worth updating the method name within `AuditablePalisadeResponse ` so it doesn't reference it as a request.

Reply:
In this case there's some options in the builder as to whether to convert the client request into a system response automatically